pkg/plugins/pfs/pfs.go

```diff
@@ -218,21 +218,25 @@ func ParsePluginFS(f fs.FS, rt *thema.Runtime) (*Tree, error) {
 		}
 		for _, s := range allslots {
 			iv := val.LookupPath(cue.ParsePath(s.slot.Name()))
-			lin, err := bindSlotLineage(iv, s.slot, r.meta, rt)
-			if lin != nil {
-				r.slotimpls[s.slot.Name()] = lin
-			}
-			if err != nil {
-				return nil, err
+			if iv.Exists() {
+				lin, err := bindSlotLineage(iv, s.slot, r.meta, rt)
```

Comment:
Can we change this slot here?

Reply:
this stuff is all about to change in the next PR (most things in pfs are rewritten), which is why i didn't bother in cleaning it up in its current state